projects/DMA-Enrico-Castellani/index.md

```diff
@@ -0,0 +1,83 @@
+--- 
+authors: 
+  - "Jeanette Knipp"
+  - "Natalie Schreiber"
+authors-url: 
+  - "https://github.com/lellahahaha"
+  - "https://github.com/natascher"
+layout: project
+title: "Deconstructing Master Artists - Enrico Castellani"
+type: project
+---
+
+![](./assets/castellani.png) 
+# Deconstructing Master Artists
```

Comment:
The Headline h1 (`#`) will be added by the system form your title. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax